sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/DocumentProducer.java

```diff
@@ -230,7 +230,7 @@ private Flux<DocumentProducerFeedResponse> feedRangeGoneProof(Flux<DocumentProdu
             }
 
             // we are dealing with Split
-            logger.info(
+            logger.debug(
```

Comment:
is this because of spark? are customers seeing too many splits? I thought the reason for this `info` or even `warn` was to make sure customers are aware that splits are happening. Since splits are once in a while event.

Reply:
Not even hit in spark (spark queries are never cross partition). Same place is hit for any cross partition query to distribute the query across producers. It is not just happening when actual splits are happening. And Diagnostics have all the necessary info to debug. This INFO level log is not needed.